x-pack/test/api_integration/apis/monitoring/elasticsearch/overview_mb.js

```diff
@@ -8,90 +8,39 @@
 import expect from '@kbn/expect';
 
 import overviewFixtureGreenPlatinum from './fixtures/overview_green_platinum.json';
-import overviewFixtureRedPlatinum from './fixtures/overview_red_platinum.json';
-import overviewFixtureShardsRelocating from './fixtures/overview_shards_relocating.json';
 import { getLifecycleMethods } from '../data_stream';
 
 export default function ({ getService }) {
   const supertest = getService('supertest');
-  const esArchiver = getService('esArchiver');
   const { setup, tearDown } = getLifecycleMethods(getService);
 
-  describe('overview mb', () => {
-    describe('with green platinum cluster', () => {
-      const archive =
-        'x-pack/test/functional/es_archives/monitoring/singlecluster_green_platinum_mb';
-      const timeRange = {
-        min: '2018-02-13T17:04:50.000Z',
-        max: '2018-02-13T17:51:55.000Z',
-      };
-
-      before('load clusters archive', () => {
-        return setup(archive);
-      });
-
-      after('unload clusters archive', async () => {
-        return tearDown();
-      });
-
-      it('should summarize elasticsearch with metrics', async () => {
-        const { body } = await supertest
-          .post('/api/monitoring/v1/clusters/fHJwISmKTFO8bj57oFBLUQ/elasticsearch')
-          .set('kbn-xsrf', 'xxx')
-          .send({ timeRange })
-          .expect(200);
-        expect(body).to.eql(overviewFixtureGreenPlatinum);
-      });
-    });
-
-    describe('with red platinum cluster', () => {
```

Comment:
We don't need these `red platinum` and `shards relocating` tests anymore?

Reply:
see https://github.com/elastic/kibana/pull/145138#discussion_r1022737625

Reply:
Just realized it's a different file but the comment applies for this one as well